.flake8

```diff
@@ -5,3 +5,8 @@ exclude = venv*,__pycache__,cache
 ignore = W503
 max-complexity = 8
 max-line-length = 120
+per-file-ignores =
+    # Ignore files because flake does not recongize 'https:' is apart of an HTML example
```

Comment:
nit: grammar, use `is a part of` or `is part of`

Reply:
Updated.